smart-contracts/contracts/CosmosBridge.sol

```diff
@@ -259,20 +282,44 @@ contract CosmosBridge is CosmosBridgeStorage, Oracle {
         );
     }
 
+    function _isManagedToken(address tokenAddress) private returns(bool) {
+        return sourceAddressToDestinationAddress[tokenAddress] != address(0);
+    }
+
+    function verifyNetworkDescriptor(uint256 _networkDescriptor) internal returns(bool) {
+        return _networkDescriptor == networkDescriptor;
+    }
+
     /**
      * @param symbol symbol of the ERC20 token on the source chain
      * @param name name of the ERC20 token on the source chain
      * @param sourceChainTokenAddress address of the ERC20 token on the source chain
      * @param decimals of the ERC20 token on the source chain
-     * @param chainDescriptor descriptor of the source chain
+     * @param _networkDescriptor descriptor of the source chain
      */
     function createNewBridgeToken(
         string calldata symbol,
         string calldata name,
         address sourceChainTokenAddress,
         uint8 decimals,
-        uint256 chainDescriptor
-    ) external onlyValidator {
+        uint256 _networkDescriptor
+    ) external onlyValidator returns(address) {
```

Comment:
We should remove this function as we only want to be creating new tokens through this function: `_submitProphecyClaimAggregatedSigs`